src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/PlaceholderTest.kt

```diff
@@ -15,8 +19,45 @@ class PlaceholderTest {
   @Test
   fun simpleTest() = uiTest {
     CommonSteps(this).createProject()
-    with(ideFrame()) {
-      assertNotNull(menu().select("A+"))
+  }
+
+  @Test
+  fun turnProjectIntoAPlusProjectTest() = uiTest {
+    CommonSteps(this).openAPlusProjectWindow()
+    step("Cancel") {
+      with(dialog("Select Course")) {
+        button("Cancel").click()
+      }
+    }
```

Comment:
As far as I understand, this test requires that `simpleTest` was run (immediately?) before it? How we are going to maintain that order?

Reply:
let's have just one test for now